.github/workflows/bundle-size.yml

```diff
@@ -1,6 +1,28 @@
 name: Compressed Size
 
-on: [pull_request]
+on:
+    pull_request:
+        paths:
```

Comment:
The problem with this config is that it prevent us from making the job "Required" on GitHub because it will make the PRs that don't need the check to get stuck forever.

The potential solution for this would be to use `if:` inside the job itself instead of `paths`

Reply:
I'm proposing that this workflow is no longer Required for PRs on GitHub.

Even if the bundle size increases exponentially, the workflow will still pass. The merging contributor must review the comparison posted in a comment on the PR and decide if the difference is acceptable before approving the PR.

Since the contributor must review the comment manually anyway, I think this is not that big of a change.

Reply:
Yes, I think it's fine for this workflow.